src/src/Helpers/ESPEasy_TouchHandler.cpp

```diff
@@ -1181,32 +1181,32 @@ bool ESPEasy_TouchHandler::plugin_webform_save(struct EventStruct *event) {
 
   uint32_t lSettings = 0u;
 
-  bitWrite(lSettings, TOUCH_FLAGS_SEND_XY,          bitRead(getFormItemInt(F("tch_events")), TOUCH_FLAGS_SEND_XY));
-  bitWrite(lSettings, TOUCH_FLAGS_SEND_Z,           bitRead(getFormItemInt(F("tch_events")), TOUCH_FLAGS_SEND_Z));
-  bitWrite(lSettings, TOUCH_FLAGS_SEND_OBJECTNAME,  bitRead(getFormItemInt(F("tch_events")), TOUCH_FLAGS_SEND_OBJECTNAME));
-  bitWrite(lSettings, TOUCH_FLAGS_ROTATION_FLIPPED, isFormItemChecked(F("tch_rotation_flipped")));
-  bitWrite(lSettings, TOUCH_FLAGS_DEDUPLICATE,      isFormItemChecked(F("tch_deduplicate")));
-  bitWrite(lSettings, TOUCH_FLAGS_INIT_OBJECTEVENT, isFormItemChecked(F("tch_init_objectevent")));
+  bitWrite(lSettings, TOUCH_FLAGS_SEND_XY,          bitRead(getFormItemInt(F("events")), TOUCH_FLAGS_SEND_XY));
+  bitWrite(lSettings, TOUCH_FLAGS_SEND_Z,           bitRead(getFormItemInt(F("events")), TOUCH_FLAGS_SEND_Z));
+  bitWrite(lSettings, TOUCH_FLAGS_SEND_OBJECTNAME,  bitRead(getFormItemInt(F("events")), TOUCH_FLAGS_SEND_OBJECTNAME));
```

Comment:
Beter store the int in a const variable before processing, as it now has to be found several times in the args